lib/mayaUsd/fileio/utils/shadingUtil.cpp

```diff
@@ -153,7 +153,11 @@ UsdShadeOutput UsdMayaShadingUtil::CreateShaderOutputAndConnectMaterial(
     TfToken               outputName = terminalName;
     SdrShaderNodeConstPtr shaderNodeDef = registry.GetShaderNodeByIdentifier(nodeID);
     if (shaderNodeDef) {
+#if PXR_VERSION >= 2025
```

Comment:
Should be `2505`. This caused an error in the preflight since it compiled with your new code.

Reply:
Thank you! Fixed